packages/components/src/navigable-container/stories/navigable-menu.tsx

```diff
@@ -1,13 +1,18 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
 /**
  * Internal dependencies
  */
 import { NavigableMenu } from '..';
 
-export default {
+const meta: ComponentMeta< typeof NavigableMenu > = {
 	title: 'Components/NavigableMenu',
 	component: NavigableMenu,
 	argTypes: {
-		children: { type: null },
+		children: { type: undefined },
```

Comment:
This doesn't allow `{ type: null }`

Reply:
I believe that the intention here was to hide the control in Storybook — that is best achieved with 

```suggestion
		children: { control: { type: null } },
```

Reply:
Thanks, committed in https://github.com/WordPress/gutenberg/pull/49377/commits/2f6b6c92a4e53cfb755d5f3645eaaf3545f7ad72